src/test/java/AnalyzerTest.java

```diff
@@ -0,0 +1,42 @@
+import junit.framework.Assert;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class AnalyzerTest {
+
+    @Test
+    public void analyzeTest1() {
+        Assert.assertEquals(false,Analyzer.analyze("a(na(lyz)e"));
+    }
+
+    @Test
+    public void analyzeTest2() {
+        Assert.assertEquals(true,Analyzer.analyze("an((aly))ze"));
```

Comment:
@DmitriyLitvin please use Assert.assertTrue
```suggestion
        Assert.assertTrue(Analyzer.analyze("an((aly))ze"));
```

Reply:
Core style? Where is space?
```suggestion
        Assert.assertEquals(true, Analyzer.analyze("an((aly))ze"));
```